region/maridia/inner-yellow/Plasma Room.json

```
@@ -388,6 +389,61 @@
       "requires": [],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 3],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            {"or": [
+              "ScrewAttack",
+              "Plasma"
```

I think the primary way that people will farm here is by shinecharging on the bottom with short enough runway to kill the bottom pirate (looks like thats a VH runway). You can then do 0-1 energy-loss sparks to kill one of the other two pirates and/or the wall pirates if you didn't get an energy drop.
```suggestion
              "Plasma",
              {"getBlueSpeed": {"usedTiles": 19, "openEnd": 0}}
```

---

The math says even if you use all 35 frames for the shinespark kill you have 92% chance to end with 20 energy. In it goes.